### PR TITLE
[REVIEW] Just adds a docs important links and avoid the message about filesystem authority not found

### PR DESCRIPTION
**Pull request by Christian8491**
This PR only avoid the logging message about `deregisterFileSystem: filesystem authority not found"` when an authority is sent, Also add a important link related with s3 stuff.

### CODE REVIEW

**Comment by williamBlazing** [comment]
Could you please verify that this resolves #987

**Comment by Christian8491** [comment]
Yes. This PR resolves it,

File: io/src/FileSystem/private/FileSystemManager_p.cpp [hunk 1]
```diff
@@ -79,8 +79,6 @@ bool FileSystemManager::Private::deregisterFileSystem(const std::string & author
 	}
 
 	if(found == false) {
```
Comment (williamBlazing) [inline io/src/FileSystem/private/FileSystemManager_p.cpp:81]: This can just be replaced by `return found;`

**Review by williamBlazing — CHANGES_REQUESTED** [review]

File: io/src/FileSystem/private/FileSystemManager_p.cpp (hunk 1 above)
Comment (Christian8491) [reply io/src/FileSystem/private/FileSystemManager_p.cpp:81]: updated.

**Comment by Christian8491** [comment]
The issue #1034 should be closed also. As we know the issue was related to key access.

**Pull request merged**